tests/tfm/tfm_psa_test/CMakeLists.txt

```diff
@@ -100,7 +100,7 @@ ExternalProject_Add(tfm_psa_arch_test_app
                 -DCROSS_COMPILE=${TFM_TOOLCHAIN_PATH}/${TFM_TOOLCHAIN_PREFIX}
                 -DPSA_TOOLCHAIN_FILE=${TFM_BINARY_DIR}/api_ns/cmake/${TFM_TOOLCHAIN_NS_FILE}
                 -DQCBOR_PATH${QCBOR_PATH_TYPE}=${CONFIG_TFM_QCBOR_PATH}
-                -DCMAKE_BUILD_TYPE=RelWithDebInfo
+                -DCMAKE_BUILD_TYPE=Debug
```

Comment:
I'm no fully sure about the consequences but is it smart to build always in Debug mode ? or only if specifically enabled?

Reply:
hmm, I suppose it is unfortunate to test the debug variant and not the production variant.

Will see what the difference is.